images/build_images

```diff
@@ -59,10 +75,26 @@ def main():
 
         if args.push:
             cmd = 'docker push {}'.format(image_name)
-            logger.debug('Running Dockerp ush command (%s) ...', cmd)
+            logger.debug('Running Docker push command (%s) ...', cmd)
             if not args.dry_run:
                 subprocess.run(cmd, shell=True, check=True)
 
 
+def _get_sdc_version(url):
+    """Parse the parcel manifest.json file to determine the SDC version corresponding to this build.  """
+    response = _get(url)
+    if not response.content:
+        logger.info('No SDC version found in manifest.json ...')
```

Comment:
Should this be `error` level or even raise an exception? How much does it break things if we can't figure out the SDC version?